packages/test-base/build.gradle.kts

```diff
@@ -215,3 +215,22 @@ kotlin {
         val iosTest by getting { dependsOn(nativeDarwinTest) }
     }
 }
+
+// FIXME Add appropriate pattern for getting test files into place for loading through the assetFile
```

Comment:
Maybe I'm missing something, but isn't this good enough? JVM/Android have fixed locations in our build system, but ios/macos move around depending on where the apps are placed?

Reply:
Yes, it works. But when ramping on this issue I thought that I could do a nice convention that work across all targets. Providing a better scheme for this should probably just be targeted in a separate issue.